steps/kotlin-script/1.0.0/step.yml

```diff
@@ -0,0 +1,41 @@
+title: Kotlin Script
+summary: |
+  Step to execute a kotlin script
+description: |
+  Step to execute a kotlin script
+website: https://github.com/jakob-grabner/bitrise-step-kotlin-script
+source_code_url: https://github.com/jakob-grabner/bitrise-step-kotlin-script
+support_url: https://github.com/jakob-grabner/bitrise-step-kotlin-script/issues
+published_at: 2023-10-18T15:02:52.092092+02:00
+source:
+  git: https://github.com/jakob-grabner/bitrise-step-kotlin-script.git
+  commit: 08ea2e8f750f6546c1da89b6387956a59173357a
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: kotlin
+  apt_get:
+  - name: kotlin
+inputs:
+- kotlin_content: |
+    println("Hello World!")
+  opts:
+    description: The Kotlin script to run
+    is_expand: false
+    is_required: true
+    summary: The Kotlin script to run
+    title: The Kotlin script
+- opts:
```

Comment:
Please make the default value of this input `$BITRISE_SOURCE_DIR`, by convention most steps use this directory as the default workdir.

Reply:
I oriented this step on the ruby script step, there it was also empty. But changed it now to BITRISE_SOURCE_DIR